{"errors":[{"message":"request to https://api.datacite.org/client-api/graphql failed, reason: Socket timeout","extensions":{"code":"INTERNAL_SERVER_ERROR"}}],"data":null}
